### ISSUE TITLE
it.util.cleanUpCode is not a function

### ISSUE THREAD

**Issue by DiegoRBaquero** [ajv-validator/ajv-keywords#129]
Hello!

On my test I have the following code:

```js
const Context = require('../contexts/context.json');
const Appointment = require('../contexts/appointment.json');

const ajv = new Ajv({
    allErrors: true,
    schemas: [
        Context,
        Appointment
    ]
});
require('ajv-keywords')(ajv, 'switch');
const validate = ajv.getSchema('context.json');
```

and it fails with:

``` 
TypeError: it.util.cleanUpCode is not a function

  › Ajv.generate_switch (node_modules/ajv-keywords/keywords/dotjs/switch.js:126:17)
  › Object.useCustomRule (node_modules/ajv/lib/compile/index.js:287:25)
  › Object.generate_custom [as code] (node_modules/ajv/lib/dotjs/custom.js:32:24)
  › generate_validate (node_modules/ajv/lib/dotjs/validate.js:382:35)
  › localCompile (node_modules/ajv/lib/compile/index.js:88:22)
  › Ajv.compile (node_modules/ajv/lib/compile/index.js:55:13)
  › Ajv._compile (node_modules/ajv/lib/ajv.js:348:27)
  › Ajv.getSchema (node_modules/ajv/lib/ajv.js:204:54)
  › Object.<anonymous> (test/contexts.js:17:22)```

Can find to understand the generated code to be able to help

**Comment by epoberezkin** [comment]
Thank you - it's the recent change, I will release the fix now.
For now, you can fix on the previous ajv version (6.12.2)

**Comment by DiegoRBaquero** [comment]
Thank you!

**Comment by epoberezkin** [comment]
@DiegoRBaquero released 3.5.1 - please let mw know if you have any issues still.

**Comment by DiegoRBaquero** [comment]
that fixed it, fastest OSS fix ever, thank you so much
